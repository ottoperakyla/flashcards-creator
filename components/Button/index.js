import React from 'react'
import styled from 'styled-components'

const Button = styled.button`
  background-color: lightblue;
  outline: 0;
  border: 0;
  padding: 0.35rem 0.5rem;
  cursor: pointer;
  font-weight:bold;
  text-transform:uppercase;
  margin: 1rem 0.15rem;
  color: white;
  display: inline-block;

  &:hover {
    color: black;
  }
`

export default props => <Button {...props} />
