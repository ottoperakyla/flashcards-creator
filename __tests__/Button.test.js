import React from 'react'
import  { create } from 'react-test-renderer'
import Button from '../components/Button'

test('snapshot', () => {
  const element = create(<Button />)
  expect(element.toJSON()).toMatchSnapshot()
})