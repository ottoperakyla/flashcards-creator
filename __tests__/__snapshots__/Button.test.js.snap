// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<button
  className="sc-bdVaJa iPNehR"
/>
`;
